### ISSUE TITLE
Direct station links result in an error

### ISSUE THREAD

**Issue by unusualbob** [jakswa/marta_ui#29]
Tried to send a link to airport station when a friend was coming in, but they got an error. Looks like a webhost configuration issue for station links.

Tried to direct link to https://marta.io/station/airport-station

```xml
<Error>
<Code>AccessDenied</Code>
<Message>Access Denied</Message>
<RequestId>0380D31335EE125A</RequestId>
<HostId> sv4xWtDcgFPGsyNj95Op58GoeqIXdWMT0ZPW/gVkmiNF9RV8BvNMezPYiaus+O4UzFOavs+kKFw= </HostId>
</Error>
```

**Comment by jakswa** [comment]
Thanks for reporting this, I got it once long ago too and never remembered to correct. I have corrected this at the host, it may take a few seconds to propagate but I'm seeing it give correct results now.

For future reference, this is reproduced by hard-refreshing on the station page (forcing it to bypass cache). If you visit normally the service-worker cache will handle things and serve up cached content.
